scripts/upload_models_to_vercel_blob.py

```
@@ -0,0 +1,83 @@
+    print(f"Uploading {filepath} ({file_size / 1024 / 1024:.1f} MB) → {blob_name} ...")
+
+    with open(filepath, "rb") as f:
+        resp = requests.put(url, headers=headers, data=f, timeout=300)
+
+    if not resp.ok:
+        print(f"ERROR {resp.status_code}: {resp.text}", file=sys.stderr)
+        sys.exit(1)
+
+    data = resp.json()
+    public_url = data.get("url")
+    print(f"\n✓ Uploaded successfully.")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)